src/hpack/hpack.py

```diff
@@ -147,10 +147,12 @@ def _to_bytes(string):
     """
     Convert string to bytes.
     """
-    if not isinstance(string, basestring):  # pragma: no cover
+    t = type(string)
+    if t is bytes:
+        return string
+    if t is not str:
         string = str(string)
-
-    return string if isinstance(string, bytes) else string.encode('utf-8')
+    return string.encode("utf-8")
```

Comment:
LGTM - since we are already touching this part: would you mind renaming the variable `string` to something less confusing, such as `data` or `value` and updating the docstring to something like "convert anything to a textual bytes representation"?

Reply:
With pleasure. That variable name threw me off so much!

Reply:
Pushed new changes